docker-compose.yml

```diff
@@ -31,20 +34,38 @@ services:
       - EXTRA_CORS_ALLOWED_HEADERS=Content-Disposition, Content-Type, x-amz-acl
       - EXTRA_CORS_EXPOSE_HEADERS=ETag
       - HOSTNAME_EXTERNAL=localstack
+    healthcheck:
```

Comment:
is the reason behind these health checks an easier/faster way to know if the container is up and running locally?

Reply:
yes this was for local dev, but for now i'm rolling back the docker updates because errors in dev